src/engine/sparqlExpressions/SparqlExpressionValueGetters.h

```diff
@@ -22,6 +22,7 @@ namespace sparqlExpression::detail {
 
 using LiteralOrIri = ad_utility::triple_component::LiteralOrIri;
 using Iri = ad_utility::triple_component::Iri;
+using IdOrLiteralOrIri = std::variant<ValueId, LiteralOrIri>;
```

Comment:
Please directly use (or alias) the `IdOrLiteralOrIri`type from `SparqlExpressionTypes.h`. That makes it much easier to refactor that type.